source/manual/review-page.html.md

```diff
@@ -26,20 +24,14 @@ We also document Icinga alerts. These explain what to do when a certain alert tr
 - Update the page
 - Remove it (don’t forget to [set up a redirect][redirects])
 - Assign it for someone else to review
-- Confirm the page is OK and set a new review date
-
-These tasks can be achieved by editing the metadata at the top of the source for that page. See the [govuk-developer-docs github repo](https://github.com/alphagov/govuk-developer-docs).
-
-### Choose a review by date
+- Confirm the page is OK and set a new 'last updated' date
```

Comment:
Could we be more specific here? I think we're saying either:

- Update the displayed timestamp by committing your edits
- Update the displayed timestamp with a no-op commit